### PR TITLE
Aziz-dev

### PR DESCRIPTION
**Pull request by azizulhasan14**
created testfile and updated readme. Please review it and comment any changes required.

### CODE REVIEW

File: src/DataTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        assertEquals(input, scheduler.getElevatorEvent());
+
+        // Validating whether all the data is read
+        assertEquals(2,floor.inputData.size());
+
+        floor.interrupt();
+        elevatorCar.start();
+        Thread.sleep(2000);
+
+        // assertFalse(scheduler.elevatorIsApproaching(2));
```
Comment (DW-Arthur) [inline src/DataTest.java:33]: delete the commented code here, and everything else LGTM

**Review by anishtankala — APPROVED** [review]
Great work overall, everything seems good!

File: src/DataTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Junit testing for the all the data to be read in the system.
+ * @author Azizul Hasan 101124159
```
Comment (DevonFruck) [inline src/DataTest.java:7]: I know you did it but we probably shouldn't be taking credit for specific parts of the program lol. Put our group number or all our names instead

File: src/DataTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ * */
+
+class DataTest{
+    
+    @Test
+    void test() throws InterruptedException{
```
Comment (DevonFruck) [inline src/DataTest.java:13]: Could we get a better name for this test function? Also the test cases should test all of the functions in each class individually.

**Review by DevonFruck — COMMENTED** [review]
I think it makes more sense to merge into my branch then into main. If you pulled all of my recent changes then I suppose its OK. We should in the future make a 'Iteration x' branch that we merge to and merge that into main when its finished

File: src/DataTest.java (hunk 2 above)
Comment (azizulhasan14) [reply src/DataTest.java]: I changed it to group

File: src/DataTest.java (hunk 3 above)
Comment (azizulhasan14) [reply src/DataTest.java]: I added individual tests to the program

**Pull request merged**